pages/404.js

```diff
@@ -0,0 +1,8 @@
+export default function fourOFour() {
```

Comment:
React components are pascal cased. To avoid confusion try `FourZeroFour` over `FourOFour`.